src/Command/Initializer/Profiles/blank/templates/public/index.php.template

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+{imports}
+
+require __DIR__ . '/../vendor/autoload.php';
+
+{defineContainer}
+
+// Set up settings
+$settings = require __DIR__ . '/../app/settings.php';
+$settings({containerVariable});
+
+// Set up dependencies
+$dependencies = require __DIR__ . '/../app/dependencies.php';
+$dependencies({containerVariable});
+
+{setContainer}
+$app = AppFactory::create();
+$callableResolver = $app->getCallableResolver();
+
+// Register middleware
+$middleware = require __DIR__ . '/../app/middleware.php';
```

Comment:
Recently via the `Slim-Skeleton` repository we sort of found that there is a flaw with using a separated middleware file due to order being important. We should keep all middleware declarations within the `index.php`. Let's move all the contents from `middleware.php` in her.

Reply:
The Blank profile creates an empty `middleware.php`, so we can simply delete it and not include in `index.php`.

Reply:
Yes! Let’s do that